README.md

```diff
@@ -1,2 +1,41 @@
 # wordpress
+
 with Docker compose
+
+## Introduction
+
```

Comment:
add more info on "why are you doing this project?" , "what benefits come with docker compose over docker", or other mentioned topics from the feedback video